disttask/framework/scheduler/manager_test.go

```diff
@@ -17,13 +17,15 @@ package scheduler
 import (
 	"context"
 	"errors"
+	"sync"
 	"testing"
 	"time"
 
+	"github.com/golang/mock/gomock"
+	"github.com/pingcap/tidb/disttask/framework/mock"
```

Comment:
we have migrated it to ```go.uber.org/mock/gomock```.

Reply:
seems no code is using this one, let's do it later